src/FailedJobMonitorServiceProvider.php

```diff
@@ -2,9 +2,9 @@
 
 namespace Spatie\FailedJobMonitor;
 
-use Illuminate\Support\ServiceProvider;
+use Illuminate\Support\ServiceProvider as ServiceProviderBase;
```

Comment:
Small nitpick: I tend to use `IlluminateServiceProvider` when I override an SP. Could you use that name as well? That's more consistent with the rest of our packages.